### PR TITLE
storage: apply same naming in metrics.go

### PR DESCRIPTION
**Pull request by gyuho**
This is PR following up with Xiang's https://github.com/coreos/etcd/pull/3795,
and to make the naming consistent with its interface change.

Please review. Thanks,

### CODE REVIEW

File: storage/metrics.go [hunk 1]
```diff
@@ -67,7 +67,7 @@ var (
 			Help:      "Total number of watchers.",
```
Comment (xiang90) [inline storage/metrics.go]: we need to change the help message too.

Comment (xiang90) [inline storage/metrics.go]: also line 66

File: storage/metrics.go [hunk 2]
```diff
@@ -67,7 +67,7 @@ var (
...
 		})
 
-	slowWatchersGauge = prometheus.NewGauge(
+	slowWatchingGauge = prometheus.NewGauge(
```
Comment (xiang90) [inline storage/metrics.go]: change line74 and 75

Comment (gyuho) [inline storage/metrics.go]: Got it. Thanks,

**Comment by xiang90** [comment]
LGTM.

Can you also add metrics to count number of watchers? Thanks!

**Comment by gyuho** [comment]
Sure. Will do. Thanks!

**Pull request merged**